aimmo-game-worker/service.py

```diff
@@ -20,7 +39,8 @@ def process_turn():
     world_map = WorldMap(**data['world_map'])
     avatar_state = AvatarState(**data['avatar_state'])
 
-    action = avatar.handle_turn(avatar_state, world_map)
+    with capture_output(logger):
+        action = avatar.handle_turn(avatar_state, world_map)
```

Comment:
if an exception is thrown, `action` will be undefined and a NameError will be throw by `action.serialise()`